MCDataProducts/inc/SimParticle.hh

```diff
@@ -51,7 +51,7 @@ namespace mu2e {
     // A default c'tor is required for ROOT.
     SimParticle():
       _id(),
-      _stageOffset(),
+      _simStage(-1u),
```

Comment:
I don't see this new data member being printed in SimParticlePrinter.

Reply:
It is there.  https://github.com/gaponenko/Offline/blob/bdba93412a46fb23377c13b2e209eedb7c1e7317/Mu2eUtilities/src/SimParticleCollectionPrinter.cc#L36

Reply:
OK, sorry I missed that.